.swiftformat

```diff
@@ -0,0 +1,13 @@
+# file options
+
+--swiftversion 5.0
+--exclude .build
+
+# format options
+
+--self insert
```

Comment:
@tomerd As mentioned in https://github.com/apple/swift-package-manager/pull/3019#discussion_r518020505, this setting is causing SwiftFormat to erroneously add `self.`. Until we report the bug and get a fix, should we reconsider this setting?

Reply:
hi @mattt in other projects (SwiftNIO, etc) we actually encouraged adding `self.` to make things are very explicit. happy to open up for discussion tho. @abertelrud @neonichu  @weissi

Reply:
I have no problem at all with explicit self, and don't care too strongly. My only concern is that running SwiftFormat with this setting currently produces invalid code.

Reply:
yes, just saw and replied in https://github.com/apple/swift-package-manager/pull/3019#discussion_r518234755